modules/bitgo/src/v2/coins/cspr.ts

```diff
@@ -106,17 +107,20 @@ export class Cspr extends BaseCoin {
    * @param {VerifyAddressOptions} params address and rootAddress to verify
    */
   isWalletAddress(params: VerifyAddressOptions): boolean {
-    const { address, rootAddress } = params;
+    const { address, keychains } = params;
```

Comment:
Looking at release 14.0.0 of `bitgo`, this looks like a breaking change in past to future usage. Think we should label this with the `BREAKING CHANGE` label?

Reply:
Based on the idea that this method should validate  that an `address` is from a wallet using the keys or in a cryptographic manner (instead of getting the `rootAddress` from the `walletDoc`), and that the keys are in the interface of the method. 
Its something being work for many coins, but im not sure if its a breaking change or its something unused yet until every coin gets implemented or refactored.

Reply:
Since it is open source, even if it isn't a breaking change because internally we update everything together, it would be a breaking change requiring external companies to update their code upon a stable release.

Reply:
Check this https://github.com/BitGo/BitGoJS/blob/master/modules/bitgo/src/v2/wallet.ts#L1497-L1523, it has a TODO, i already made changes to 2 coins before this one and there are other coins that need implementation or refactor.
I think its a method only used internally, but its not private atm.